projects/demo-integrations/cypress/support/visit.ts

```diff
@@ -112,7 +112,7 @@ export function tuiVisit(path: string, options: TuiVisitOptions = {}): void {
 
     cy.wait(DEFAULT_TIMEOUT_AFTER_PAGE_REDIRECTION);
 
-    // cy.get('app._loaded').should('exist'); // uncomment when next-app will have this class
+    cy.get('app._loaded').should('exist');
```

Comment:
let's try to remove `wait` somewhere

Reply:
I've removed all `DEFAULT_TIMEOUT_AFTER_PAGE_REDIRECTION`.

Workflows have successfully finished 3 time in a row ✅